src/main/java/Entities/Player.java

```diff
@@ -79,20 +114,44 @@ public StringBuilder getPossibleActions() {
         return actions;
     }
```

Comment:
We might want to remove this chunk of code later: getting the possible actions is no longer handled by this class

Reply:
Yeah, let's leave it commented for now just in case but most likely we will remove this soon.